docs/resources/oxide_instance.md

```diff
@@ -6,7 +6,7 @@ page_title: "oxide_instance Resource - terraform-provider-oxide"
 
 This resource manages instances.
 
--> Boot disk, disk attachment, and network interface updates will stop and reboot the instance.
+-> Updates will stop and reboot the instance.
```

Comment:
Is there any update that won't stop and reboot the instance?

Reply:
Nah, they all do